sdk/core/core/inc/az_span.h

```diff
@@ -55,7 +55,7 @@ AZ_NODISCARD AZ_INLINE int32_t az_span_capacity(az_span span) { return span._int
  * @brief Creates an empty span
  *
  */
-AZ_NODISCARD AZ_INLINE az_span az_span_null() { return (az_span){ 0 }; }
+#define AZ_SPAN_NULL (az_span){ ._internal.ptr = NULL, ._internal.length = 0, ._internal.capacity = 0 }
```

Comment:
Nit: you can initialize as (az_span){ ._internal = {.ptr = NULL, .length = 0, .capacity = 0 }}

Reply:
And also I think you need to include <stddef.h> for NULL

Reply:
Thanks for applying my feedback!